packages/sanity/src/core/studio/components/navbar/free-trial/FreeTrialButton.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components'
+import {Button, Text, Card, Stack} from '@sanity/ui'
+import {BoltIcon} from '@sanity/icons'
+import {purple, yellow} from '@sanity/color'
+import {useTranslation} from 'react-i18next'
+
+const StyledButton = styled(Button)<{smallIcon: boolean}>`
+  padding: ${(props) => (props.smallIcon ? '1px' : 0)};
+  position: relative;
+`
+
+const CenteredStroke = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`
+
+const SvgFilledCircle = ({daysLeft}: {daysLeft: number}) => {
+  const totalDays = 30
```

Comment:
Unfortunately we can't rely on the total days being 30 as we have a boosted growth plan that is 90 days. Likely best to send back the total days in the backend response

Reply:
Updated staging backend just now to send `totalTrialDays` in response

Reply:
Cool, thanks @svirs .

Reply:
Added support for it